pkg/token/server.go

```diff
@@ -165,7 +166,7 @@ func postAccessToken(ts *tokenService, w http.ResponseWriter, r *http.Request) {
 	}
 
 	// create cache key
-	k := CacheKey{Domain: domain, Role: role}
+	k := CacheKey{Domain: domain, Role: role, WriteFileRequired: false}
```

Comment:
no need to declare at this point:
```suggestion
	k := CacheKey{Domain: domain, Role: role}
```

Reply:
fixed in: https://github.com/AthenZ/k8s-athenz-sia/pull/131/commits/65185656cd52eeb18ad050fd0a5a25a5033b61e0